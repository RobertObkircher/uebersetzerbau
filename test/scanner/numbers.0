2983 (3290239 $ab1-983434 -4383 (3293)
