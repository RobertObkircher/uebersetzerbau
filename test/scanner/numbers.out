num 2983
(
num 3290239
num 2737
-
num 983434
-
num 4383
(
num 3293
)
